### PR TITLE
Portfolio Contact section Mobile

### PR DESCRIPTION
**Pull request by mudasiri**
**Portfolio Website Mobile First Design Contact Form Section**
In this Phase i designed the contact form section of my portfolio website in collaboration with my partner @SabaAhmad786  we  include the following sections:

-  Title Section
-  Brief description
-  Contact Form
- We used flex box 

The entire code is designed for mobile view only

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -139,21 +139,24 @@ <h2>Skills</h2>
...
-        <form action="" id="contact-form">
-            <input type="text" value="Mudasir">
-            <input type="email" name="" id="" value="muda@nonihub.org">
-            <textarea name="" id="" cols="30" rows="10" form="contact-form" placeholder="Write your message here"></textarea>
-            <input class="submit-btn" type="submit" value="Get In Touch">
-        </form>
-        </div>
+    <section class="contact-area">
+        <div class="contact-me">
+            <h1>Contact Me</h1>
```
Comment (mgmediaweb) [inline index.html:147]: Kindly, for accessibility reasons, your page can only contain 1 header H1, I suggest you change it to another type, H2, H3, etc.

File: index.html [hunk 2]
```diff
@@ -139,21 +139,24 @@ <h2>Skills</h2>
...
+            <p>
+                If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+            </p>
+           
+                <form action="https://formspree.io/f/mbjbopyj" method="post">
+                    <input type="text" id="name" name="name" required minlength="1" maxlength="30" placeholder="SabaAhmad and Muda Issah" required>
```
Comment (mgmediaweb) [inline index.html:153]: Kindly, I suggest you remove the repeated "required" attribute

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @mudasiri , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost1

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (tresorsawasawa) [inline index.html:147]: - [ ] As mentioned by the previous reviewer, kindly make sure you use a `<hx>` tag other than `<h1>`  (e.g.: `<h2>`, `<h3>`,...) for semantics and accessibility reasons. Your project is supposed to only have one `<h1>`.

File: index.html (hunk 2 above)
Comment (tresorsawasawa) [inline index.html:153]: - [ ] As mentioned by the previous reviewer, kindly make sure you remove one `required` attribute in your `input` tag as it should have only one `required` and not **two** attributes. I know you can do that :muscle: :muscle:

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @mudasiri 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @mudasiri , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
imageswelljob

## Highlights
- [ ] All linters are green
- [ ] All changes requests are implemented.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**